pull-dependencies

```diff
@@ -11,13 +11,24 @@
 $version = '2.0'
 $isPublic = true
 
+# BEGIN_HIDE
```

Comment:
Do we need BEGIN_HIDE any more now that there's no public vs. private repo distinction?